examples/validators/policy-controller-validate/invalid-configmap/.expected/results.yaml

```diff
@@ -0,0 +1,14 @@
+items:
+- message: |-
+    The following banned keys are being used in the config map: {"private_key"}
```

Comment:
@droot @Shell32-Natsu We need to think about what the kpt CLI output looks like for functions emitting structured results. It wouldn't be sufficient to just display the message string as it does not contain enough information unless the user looks at the FunctionResults.

Reply:
Related: How's the non-structured output being tested?

Reply:
> @droot @Shell32-Natsu We need to think about what the kpt CLI output looks like for functions emitting structured results. It wouldn't be sufficient to just display the message string as it does not contain enough information unless the user looks at the FunctionResults.

Yes, this has come up while writing the spec for FunctionResult and we didn't have a very good answer. I will schedule sometime tomorrow for us to go over this.